package.json

```diff
@@ -87,7 +87,7 @@
     "@puppeteer/browsers": "^1.4.1",
     "@rollup/plugin-node-resolve": "^15.0.1",
     "@rollup/plugin-terser": "^0.4.0",
-    "chalk": "^5.2.0",
+    "chalk": "^5.3.0",
```

Comment:
Do you mind avoid updating `package.json` (and `package-lock.json`) in this PR?

Reply:
Fixed